Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx

```diff
@@ -77,15 +77,17 @@ export default class EquipmentContainer extends React.Component<IProps, IState>
 
         const commonAttributeKeys = await this.context.fetch(attrFetchUrl);
 
-        const equipmentTypeFetchUrl = `/api/${this.context.team.slug}/equipment/ListEquipmentTypes/`;
+        const equipmentTypeFetchUrl = `/api/${
+            this.context.team.slug
+        }/equipment/ListEquipmentTypes/`;
```

Comment:
Don't like the white space, but doesn't make a difference

Reply:
blame prettier, not me 🤷‍♀